config/certmanager/kustomization.yaml

```diff
@@ -3,3 +3,5 @@ resources:
 
 configurations:
 - kustomizeconfig.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
Can we put these at the top of the file?

Reply:
Same everywhere else

Reply:
sure can